enterprise/models.py

```diff
@@ -3955,6 +3955,15 @@ class Meta:
         )
     )
 
+    marked_authorized = models.BooleanField(
+        blank=False,
+        null=False,
+        default=False,
+        help_text=_(
+            "If the edX Service Provider metadata was authorized from the Identity Provider side."
```

Comment:
maybe make a comment that it's the admin indicating the service provider metadata was uploaded- it's come back to bite us in the past when the admin mindlessly checks the box and doesn't actually do the thing we're asking and we don't want to confuse any devs/ecs that the actions been done... more so that they TOLD us it was done lol